charts/logging/loki/values.yaml

```diff
@@ -64,72 +51,87 @@ loki:
         index:
           prefix: index_
           period: 24h
-    server:
-      http_listen_port: 3100
+      - from: 2022-01-11
+        store: boltdb-shipper
+        object_store: filesystem
+        schema: v12
+        index:
+          prefix: loki_index_
+          period: 24h
```

Comment:
Upstream chart now lists this schema, I more or less blindly assume we have to also add it to our list?

Reply:
We use it in MLA as well.